packages/eslint-plugin/docs/rules/no-floating-promises.mdx

```diff
@@ -115,6 +115,54 @@ await (async function () {
 })();
 ```
 
+### `allowForKnownSafePromises`
+
+This option allows marking specific types as "safe" to be floating. For example, you may need to do this in the case of libraries whose APIs return Promises whose rejections are safely handled by the library.
+
+Each item must be one of:
```

Comment:
For a first-time reader, this needs a bit of clarification... Each item of what? 
Consider something like the following.

_This option takes an array of type specifier objects. There are three possible types of specifiers: "file", "lib", and "package"._

followed by the list of what each does

Reply:
I just copied what was written here ... https://typescript-eslint.io/rules/prefer-readonly-parameter-types#allow 
Should I change both or only this one ?

Reply:
Oh, thanks for pointing that out! I would say, in that case, leave it as is 🙂 Better if they both get looked at in a separate issue, then, rather than being reexamined in this PR